manifest.go

```diff
@@ -407,3 +559,15 @@ func (m *Manifest) RequiredPackages() map[string]bool {
 
 	return mp
 }
+
+// PruneOptionsFor returns the prune options for the passed project root.
+//
+// It will return the root prune options if the project does not have specific
+// options or if it does not exists in the manifest.
+func (m *Manifest) PruneOptionsFor(pr gps.ProjectRoot) gps.PruneOptions {
```

Comment:
Do we need this function? I don't see it being used anywhere. Maybe for future usage?

Reply:
Yes. The other PR will use this function.